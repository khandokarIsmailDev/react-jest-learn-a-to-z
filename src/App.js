import logo from './logo.svg';
import './App.css';
import Logo from './logo.svg'
import { useState } from 'react';
import Button from './Button';
import Users from './Users';
import Func from './Func';

function App() {

  

  return (
    <div className="App">
      <p>Learn React</p>
      <h1>This is Heading Tag1u</h1>
      <input type='text' placeholder='Enter User Name' name='username' id='userId' /><br/><br/>
      <img
        title='this is react oficial logo'
        src={Logo}
        className="img-fluid rounded-top App-logo"
        alt=""
      />
      <h2>I Am ISMAIL</h2><br/>

      <Button/>

      <Users/>

      <Func/>

    </div>
  );
}

export default App;
