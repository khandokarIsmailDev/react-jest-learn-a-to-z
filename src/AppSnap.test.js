import { render } from "@testing-library/react";
import App from "./App";

test("ar maddome snapshot create korbo",()=>{
    const container = render(<App/>) 
    expect(container).toMatchSnapshot()
})