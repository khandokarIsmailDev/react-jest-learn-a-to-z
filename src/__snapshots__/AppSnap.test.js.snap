// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ar maddome snapshot create korbo 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <p>
          Learn React
        </p>
        <h1>
          This is Heading Tag1u
        </h1>
        <input
          id="userId"
          name="username"
          placeholder="Enter User Name"
          type="text"
        />
        <br />
        <br />
        <img
          alt=""
          class="img-fluid rounded-top App-logo"
          src="logo.svg"
          title="this is react oficial logo"
        />
        <h2>
          I Am ISMAIL
        </h2>
        <br />
        <div
          class="App"
        >
          <h1>
            Test Click Event with button
          </h1>
          <button>
            Update Data
          </button>
          <h3 />
        </div>
        <div>
          <h2>
            user list
          </h2>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <p>
        Learn React
      </p>
      <h1>
        This is Heading Tag1u
      </h1>
      <input
        id="userId"
        name="username"
        placeholder="Enter User Name"
        type="text"
      />
      <br />
      <br />
      <img
        alt=""
        class="img-fluid rounded-top App-logo"
        src="logo.svg"
        title="this is react oficial logo"
      />
      <h2>
        I Am ISMAIL
      </h2>
      <br />
      <div
        class="App"
      >
        <h1>
          Test Click Event with button
        </h1>
        <button>
          Update Data
        </button>
        <h3 />
      </div>
      <div>
        <h2>
          user list
        </h2>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
